pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -561,6 +561,10 @@ in
     dependencies = with self; [ copilot-lua plenary-nvim ];
   };
 
+  copilot-lualine = super.copilot-lualine.overrideAttrs {
+    dependencies = with self; [ copilot-lua lualine-nvim ];
+  };
+
```

Comment:
Can we try to throw a
```
doInstallCheck ) true;
nvimRequireCheck = "copilot-lua";
```
here ?

Reply:
We can. the nvimRequireCheck should have only copilot-lua or also lualine-nvim? I'm not familiar with this part here

Reply:
It should be `copilot-lualine` sorry. No need to check the dependencies.

Reply:
please check